pages/api/auth/signup.ts

```diff
@@ -69,5 +69,17 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     },
   });
 
+  // If user has been invitedTo a team, we accept the membership
```

Comment:
@miguelnietoa may tests be added?

Reply:
Of course, I can add some expectations in the Successful signup test. Working on that now!